backend/src/main/java/org/cryptomator/hub/entities/RemoveVaultMembershipEvent.java

```diff
@@ -0,0 +1,53 @@
+package org.cryptomator.hub.entities;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.DiscriminatorValue;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+
+import java.time.Instant;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Table(name = "remove_vault_membership_event")
+@DiscriminatorValue(RemoveVaultMembershipEvent.TYPE)
+public class RemoveVaultMembershipEvent extends AuditEvent {
```

Comment:
Looking at the wording, this could be imroved by renaming it to `RemoveVaultMemberEvent` now. Similarly, `AddVaultMembershipEvent`  to  `AddVaultMemberEvent`.

Reply:
I just noticed, this would trigger renamings at some places (database, frontend). If this is a major blocker, say it and we ignore this request.

Reply:
Fixed in 0df8420076f892e01e91ddb372e795ac40ee0c63.